sdk/go/internal/querybuilder/querybuilder.go

```diff
@@ -110,7 +110,16 @@ func (s *Selection) unpack(data interface{}) error {
 		if i.alias != "" {
 			k = i.alias
 		}
-		data = data.(map[string]interface{})[k]
+
+		// Try to asert type of the value
+		switch f := data.(type) {
+		case map[string]interface{}:
+			data = data.(map[string]interface{})[k]
```

Comment:
nit:
```suggestion
			data = f[k]
```

Reply:
Fixed in https://github.com/dagger/dagger/pull/4683/commits/2001712417162fca4656a559ddaadebbf73b1274